streams/src/test/java/org/apache/kafka/streams/processor/internals/SourceNodeTest.java

```diff
@@ -112,6 +115,14 @@ private void shouldExposeProcessMetrics(final String builtInMetricsVersion) {
             final String parentGroupName = "stream-task-metrics";
             assertTrue(StreamsTestUtils.containsMetric(metrics, "process-rate", parentGroupName, metricTags));
             assertTrue(StreamsTestUtils.containsMetric(metrics, "process-total", parentGroupName, metricTags));
+
+            final String sensorNamePrefix = "internal." + threadId + ".task." + context.taskId().toString();
+            final Sensor processSensor =
+                metrics.getSensor(sensorNamePrefix + ".node." + context.currentNode().name() + ".s.process");
+            assertThat(
+                processSensor.parents().stream().map(Sensor::name).collect(Collectors.toList()),
+                contains(sensorNamePrefix + ".s.process")
+            );
```

Comment:
This verification tests the correct wiring of the node-level sensor with its task-level sensor.